Jenkinsfile

```diff
@@ -914,7 +914,7 @@ pipeline {
                                                         name: 'Fault injection leaks',
                                                         id: 'NLT_client')]
                             junit testResults: 'nlt-junit.xml'
-                            archiveArtifacts artifacts: 'nlt_logs/centos7.fault-injection/'
+                            archiveArtifacts artifacts: 'nlt_logs/centos8.fault-injection/'
                         }
                     }
                 } // stage('Fault inection testing')
```

Comment:
Should these match?
```suggestion
                } // stage('Fault inection testing on CentOS 8')
```

Reply:
Probably, but also a comment that can be cleaned up in a subsequent PR (and might be in a PR I have in progress even).